apps/trading-e2e/src/integration/market-order.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Market orders
+
+  Scenario Outline: Successfull market buy orders
+    Given I am on the homepage
+    And I connect to Vega Wallet
+    And I navigate to markets page
+    When I click on active market
+    And place a buy '<marketOrderType>' market order
+    Then order request is sent
+
+    Examples:
+      | marketOrderType |
+      | FOK             |
+      | IOC             |
+
+  Scenario Outline: Successfull Limit buy orders
+    Given I am on the homepage
+    And I connect to Vega Wallet
+    And I navigate to markets page
+    When I click on active market
+    And place a buy '<limitOrderType>' limit order
+    Then order request is sent
+
+    Examples:
+      | limitOrderType |
+      | IOC            |
+      | FOK            |
+      | GTT            |
+      # | GFA            |
```

Comment:
Can we have a comment explaining why this one is not possible please

Reply:
Comment left